### PR TITLE
[type: refactor] optimize log config for dubbo subscribe warning message.

### PR DESCRIPTION
**Pull request by qicz**

### CODE REVIEW

File: shenyu-bootstrap/src/main/resources/application-local.yml [hunk 1]
```diff
@@ -144,4 +144,9 @@ logging:
     org.apache.shenyu.bonuspoint: info
     org.apache.shenyu.lottery: info
     org.apache.shenyu: info
+    # reset the com.alibaba log
+    com.alibaba.cloud.dubbo.autoconfigure.DubboServiceDiscoveryAutoConfiguration: warn
```
Comment (yu199195) [inline shenyu-bootstrap/src/main/resources/application-local.yml:148]: we have this jar?

Comment (qicz) [reply shenyu-bootstrap/src/main/resources/application-local.yml:148]: when using the spring cloud Alibaba the jar exists. the config is remove the many warning message

**Comment by qicz** [comment]
**the warning message**

```bash
2021-12-16 14:27:54 [Nacos-Watch-Task-Scheduler-1] WARN  com.alibaba.cloud.dubbo.metadata.repository.DubboServiceMetadataRepository - Current application will subscribe all services(size:1) in registry, a lot of memory and CPU cycles may be used, thus it's strongly recommend you using the externalized property 'dubbo.cloud.subscribed-services' to specify the services
2021-12-16 14:28:24 [Nacos-Watch-Task-Scheduler-1] WARN  com.alibaba.cloud.dubbo.metadata.repository.DubboServiceMetadataRepository - Current application will subscribe all services(size:1) in registry, a lot of memory and CPU cycles may be used, thus it's strongly recommend you using the externalized property 'dubbo.cloud.subscribed-services' to specify the services
2021-12-16 14:28:54 [Nacos-Watch-Task-Scheduler-1] WARN  com.alibaba.cloud.dubbo.metadata.repository.DubboServiceMetadataRepository - Current application will subscribe all services(size:1) in registry, a lot of memory and CPU cycles may be used, thus it's strongly recommend you using the externalized property 'dubbo.cloud.subscribed-services' to specify the services
2021-12-16 14:29:24 [Nacos-Watch-Task-Scheduler-1] WARN  com.alibaba.cloud.dubbo.metadata.repository.DubboServiceMetadataRepository - Current application will subscribe all services(size:1) in registry, a lot of memory and CPU cycles may be used, thus it's strongly recommend you using the externalized property 'dubbo.cloud.subscribed-services' to specify the services
2021-12-16 14:29:54 [Nacos-Watch-Task-Scheduler-1] WARN  com.alibaba.cloud.dubbo.metadata.repository.DubboServiceMetadataRepository - Current application will subscribe all services(size:1) in registry, a lot of memory and CPU cycles may be used, thus it's strongly recommend you using the externalized property 'dubbo.cloud.subscribed-services' to specify the services
2021-12-16 14:30:24 [Nacos-Watch-Task-Scheduler-1] WARN  com.alibaba.cloud.dubbo.metadata.repository.DubboServiceMetadataRepository - Current application will subscribe all services(size:1) in registry, a lot of memory and CPU cycles may be used, thus it's strongly recommend you using the externalized property 'dubbo.cloud.subscribed-services' to specify the services

```

**Comment by qicz** [comment]
funny

**Pull request closed without merging**